EasyPost/Tracker.cs

```diff
@@ -102,7 +102,12 @@ public static Tracker Retrieve(string id)
         public static bool CreateList(Dictionary<string, object> param)
         {
             Request request = new Request("trackers/create_list", RestSharp.Method.POST);
-            request.AddBody(new Dictionary<string, object>() { { "trackers", param } });
+            request.AddBody(new Dictionary<string, object>
+            {
+                {
+                    "trackers", param
```

Comment:
`param` should be pluralized.

Reply:
Resolved in [e01e5b4](https://github.com/EasyPost/easypost-csharp/pull/205/commits/e01e5b4a785997e20d948c2c0fccb4d2a0f0747b)